diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
index 3b9bf87..b068e25 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
@@ -142,14 +142,8 @@ const BottomSheetBackdropComponent = ({
       pointerEvents={pointerEvents}
       accessible={_providedAccessible ?? undefined}
       accessibilityRole={_providedAccessibilityRole ?? undefined}
-      accessibilityLabel={_providedAccessibilityLabel ?? undefined}
-      accessibilityHint={
-        _providedAccessibilityHint
-          ? _providedAccessibilityHint
-          : `Tap to ${
-              typeof pressBehavior === 'string' ? pressBehavior : 'move'
-            } the Bottom Sheet`
-      }
+      accessibilityLabel="Sluiten"
+      accessibilityHint="Dubbeltik om te sluiten"
     >
       {children}
     </Animated.View>
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
index 3f165dd..7536f01 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackground/BottomSheetBackground.tsx
@@ -9,9 +9,7 @@ const BottomSheetBackgroundComponent = ({
 }: BottomSheetBackgroundProps) => (
   <View
     pointerEvents={pointerEvents}
-    accessible={true}
-    accessibilityRole="adjustable"
-    accessibilityLabel="Bottom Sheet"
+    accessible={false}
     style={[styles.container, style]}
   />
 );
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/constants.ts b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/constants.ts
index 98d76c1..79601d5 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/constants.ts
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/constants.ts
@@ -1,8 +1,7 @@
 const DEFAULT_ACCESSIBLE = true;
 const DEFAULT_ACCESSIBILITY_ROLE = 'adjustable';
-const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom sheet handle';
-const DEFAULT_ACCESSIBILITY_HINT =
-  'Drag up or down to extend or minimize the bottom sheet';
+const DEFAULT_ACCESSIBILITY_LABEL = 'Sluiten';
+const DEFAULT_ACCESSIBILITY_HINT = 'Veeg omlaag om te sluiten';
 
 export {
   DEFAULT_ACCESSIBLE,
