diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
index 35597ce..54a6048 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx
@@ -121,10 +121,8 @@ const BottomSheetBackdropComponent = ({
         pointerEvents={pointerEvents}
         accessible={true}
         accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessibilityLabel="Bottom Sheet sluit knop"
+        accessibilityHint="Klik om de bottom sheet te sluiten"
       >
         {children}
       </Animated.View>
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandleContainer/BottomSheetHandleContainer.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandleContainer/BottomSheetHandleContainer.tsx
index 2219e0f..efcbd69 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandleContainer/BottomSheetHandleContainer.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandleContainer/BottomSheetHandleContainer.tsx
@@ -94,8 +94,8 @@ function BottomSheetHandleContainerComponent({
         key="BottomSheetHandleContainer"
         accessible={true}
         accessibilityRole="adjustable"
-        accessibilityLabel="Bottom Sheet handle"
-        accessibilityHint="Drag up or down to extend or minimize the Bottom Sheet"
+        accessibilityLabel="Bottom Sheet trigger"
+        accessibilityHint="Veeg omlaag om terug te gaan naar het vorige scherm"
         onLayout={handleContainerLayout}
       >
         <HandleComponent
