diff --git a/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx b/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx
index 9f6aaf0..68e36a3 100644
--- a/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx
+++ b/node_modules/react-native-swiper-flatlist/src/components/SwiperFlatList/SwiperFlatList.tsx
@@ -233,7 +233,7 @@ export const SwiperFlatList = React.forwardRef(
       },
     ]);
 
-    const flatListProps: FlatListProps<unknown> & { ref: React.RefObject<RNFlatList<unknown>> } = {
+    const flatListProps: FlatListProps<unknown> & { ref: React.RefObject<RNFlatList<unknown> | null> } = {
       scrollEnabled,
       ref: flatListElement,
       keyExtractor: (_item, _index) => {
